### ISSUE TITLE
Exit errors in background namespace generate a unique error every time

### ISSUE THREAD

**Issue by tombruijn** [appsignal/appsignal-elixir#76]
This causes some 11k errors to be reported for one user (along with as many notifications 😖 ).

Reported reproducible code:

```elixir
$ iex -S mix
> Task.async(fn -> Process.sleep(1000) end) |> Task.await(500)
```

Cannot reproduce on 0.9.2 (which user uses) or 0.10.0

As reported in https://app.intercom.io/a/apps/yzor8gyw/inbox/conversation/7786937331

**Comment by arjan** [comment]
This is how it's reproducible:

```
:proc_lib.spawn_link(fn -> Task.async(fn -> Process.sleep(1000) end) |> Task.await(500) end)
```

It indeed generates a new error message each time. This is because the process id is part of the error reason string, and the pid is different each time.

The best way to fix this is probably replacing the `#PID<0.32.11>` parts of the string with something like `#PID<...>`.

**Comment by tombruijn** [comment]
Fixed by #78
